src/frontend/TabChanges/utils.js

```diff
@@ -0,0 +1,6 @@
+export default function changeDisplayName(change) {
```

Comment:
may be its better to keep the `utils` in the `frontend` instead of `frontend/tabChanges/`